parser/parser_test.go

```diff
@@ -72,7 +72,8 @@ func TestParser_ParseStatements(t *testing.T) {
 }
 
 func TestParser_Format(t *testing.T) {
-	for _, dir := range []string{"./testdata/dml", "./testdata/ddl", "./testdata/query", "./testdata/basic"} {
+	for _, dir := range []string{"./testdata/ddl"} {
```

Comment:
@sharadgaur We should revert this change?

Reply:
ah let me fix it